src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala

```diff
@@ -16,9 +31,9 @@ object InputRecord {
   def parse(file: File): Try[Stream[InputRecord]] = Try {
     CSVParser.parse(file, Charsets.UTF_8, CSVFormat.RFC4180)
       .asScala
-      .filter(_.asScala.nonEmpty)
-      .drop(1)
+      .drop(1) // Ingore the header
```

Comment:
The `filter`-didn't do anything. I assume it was intended as the `withFilter`-call below.